lisp_ease/CMakeLists.txt

```diff
@@ -0,0 +1,199 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(lisp_ease)
```

Comment:
Rename whole ROS package / asdf system / Lisp package into cram_knowrob_vr / cram-knowrob-vr / cram-knowrob-vr (nicknames knowrob-vr or kvr). Put this package under cram_knowrob.